pkg/status/templates/orchestrator.tmpl

```diff
@@ -0,0 +1,67 @@
+{{/*
+*/}}=====================
+Orchestrator Explorer
+=====================
+
+{{- if .Error }}
+  Error: {{ .Error }}
+{{- end }}
+{{- if .Disabled }}
+  Disabled: {{ .Disabled }}
+{{- end }}
+{{- if .ClusterIDError }}
+  ClusterIDError: {{.ClusterIDError}}
+{{- end }}
+
+{{- if .ClusterID }}
+  ClusterID: {{.ClusterID}}
+{{- end }}
+{{- if .ClusterNameError }}
+  ClusterIDError: {{.ClusterNameError}}
+{{- end }}
+
+{{- if .ClusterName }}
+  ClusterName: {{.ClusterName}}
+{{- end }}
+{{- if .ContainerScrubbing }}
+  {{.ContainerScrubbing}}
+{{- end}}
+
+  ======================
+  Orchestrator Endpoints
+  ======================
+{{- if .OrchestratorEndpoint }}
+    OrchestratorEndpoint: {{.OrchestratorEndpoint}}
+{{- end }}
+{{- if .OrchestratorAdditionalEndpoints }}
+    OrchestratorAdditionalEndpoints: {{.OrchestratorAdditionalEndpoints}}
+{{- end }}
+
+  ===============
+  Forwarder Stats
```

Comment:
Yes, makes sense. I can remove them 👍 . Need to doublecheck whether this change propagates other things though 🤔 .

Reply:
Looking into the code again this is more work because: 
Forwarder stats are defined as expvar and used as a whole during template rendering.

If we would want to remove them from the output we would either have to:
- remove them from transaction (the upper expvar): https://github.com/DataDog/datadog-agent/blob/0f440aac2c3f24722bee1f2a2f12106f4719e80a/pkg/forwarder/forwarder.go#L111-L117 (1)
  - create our own expvarMap and make sure that `pods` are still shown on the agent status output, because the process-agent is still sending pods workload 

- modify the dca status output to remove the orchestrator related workloads (2)
  - `removeOrchestratorPartsFromForwarder(stats[forwarderStats])

I think it is something we can do in a follow up with a ticket.

Reply:
yeah i thought that might be how it is. Ok we'll do it in a follow up that's fine 👍